api/v1/types.go

```diff
@@ -16,6 +16,23 @@ const (
 	StateTypeInitializing StateType = "Initializing"
 )
 
+const (
+	checkMark   = "\033[32m✔\033[0m"
+	warningSign = "\033[31m✘\033[0m"
+)
+
+// TerminalHeader returns a terminal header based on its health state.
+func (st StateType) TerminalHeader() string {
```

Comment:
It's not recommended that define the method for StateType here.  We should define it close it caller and IMO there is only one package(scan) relied on it.

Reply:
removed